### DISCUSSION TITLE
issure how to search based on the actual range created

### DISCUSSION THREAD

**Discussion by busy-tester** [python-gitlab/python-gitlab#2269] [Q&A] [answered]
`issues = project.issues.list(labels='kind/bug', state='closed',create=)`
I want to filter by the time range created, how do I do that?

**Comment by JohnVillalovos** [comment, marked as answer]
I recommend looking at the upstream docs:

https://docs.gitlab.com/ee/api/api_resources.html

In particular: https://docs.gitlab.com/ee/api/issues.html#list-issues
